### DISCUSSION TITLE
LResponse como pegar o retorno do OctetStream

### DISCUSSION THREAD

**Discussion by ebnezerjrs** [viniciussanchez/RESTRequest4Delphi#168] [General]
Gostaria de pegar o retorno desse end point abaixo pelo Lresponse
Tipo:

LResponse := TRequest.New.BaseURL(edtBaseURL.Text)
                   .Accept(edtAccept.Text)
                   .AddParam('cnpj', '82960631404')
                   .AddParam('path', 'D:\Projetos\D11.3\VCL\Nexus-ERP-Fiscal-4.3\Sistema\OS\82960631404\Imagens\OS-10-item-7.png')
                   .BasicAuthentication('nexusapi','nexusapi')
                   .Adapters(TDataSetSerializeAdapter.New(MemTable))
                   .Get;

![image](https://github.com/viniciussanchez/RESTRequest4Delphi/assets/95051045/07ee38eb-f477-4c73-ab00-f69abd4e2a8a)

**Comment by viniciussanchez** [comment]
Só usar o LResponse.ContentStream

**Reply by ebnezerjrs** [reply]
VC TEM UM EXEMPLO COMO USO **LResponse.ContentStream** ?
